controllers/admin/AdminConfigurePaymentExampleController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+class AdminConfigurePaymentExampleController extends ModuleAdminController
+{
+    public function __construct()
+    {
+        $this->bootstrap = true;
+        $this->className = 'Configuration';
+        $this->table = 'configuration';
+
+        parent::__construct();
+
+        if (empty(Currency::checkPaymentCurrencies($this->module->id))) {
+            $this->warnings[] = $this->l('No currency has been set for this module.');
```

Comment:
```suggestion
            $this->warnings[] = $this->l('No currency has been set for this module.');
```

Is `->l` the best way to handle translations in the recent PrestaShop version?

Reply:
Module translations with new translation component works only since 1.7.7.0 I guess
We want an example for partners working for all PrestaShop 1.7 versions